gpkit/constraints/set.py

```diff
@@ -5,12 +5,6 @@
 from ..repr_conventions import _str, _repr, _repr_latex_
 
 
-def _var_sort_key(var):
-    "return thing to compare for Variable sorting"
-    k = var.key
-    return (k.str_without(["units", "idx"]), k.idx)
-
-
```

Comment:
this fixed #724; I want to retain some functionality (not necessarily `__getitem__` for getting a correctly-sorted VectorVariable from a model.